mission_summary/mission_summary.py

```diff
@@ -0,0 +1,209 @@
+    table = tab_ax.table(cellText=df.values, colLabels=df.columns, cellLoc='center')
+    table.scale(2, 2)
+
+    for (row, col), cell in table.get_celld().items():
+        if (row == 0) or (col == -1):
+            cell.set_text_props(fontproperties=FontProperties(weight='bold'))
+    for (row, col), cell in table.get_celld().items():
+        if (col == 0):
+            cell.set_text_props(fontproperties=FontProperties(weight='bold'))
+
+    # Basic checks
+    ax.text(-0.05, 0.20, 'Basic checks', transform=ax.transAxes, fontsize=font_size, weight='bold')
+    basictext = '\n'.join((
+        f"Issues with profile number: {phrase_numberprof_check(ds)} ",
+        f"Issues with profile duration: {phrase_duration_check(ds)}",))
+    ax.text(-0.05, 0.19, basictext, transform=ax.transAxes, fontsize=font_size,
+            verticalalignment='top')
+    ## Basic checks table
+    df2 = pd.DataFrame({'': ['Global range', 'Spike test', 'Flat test', 'Hysteresis', 'Drift'],
+                        'Temperature': ['✓', '✓', '✓', '✓', '✓'],
+                        'Salinity': ['✓', '✓', '✓', '✓', '✓'],
+                        'Oxygen': ['✓', '✓', '✓', '✓', '✓'],
+                        'Chlorophyll': ['✓', '✓', '✓', '✓', '✓']})
+
+    tableT = fill_tableqc(df2,ds, var='TEMP')
+    tableS = fill_tableqc(tableT,ds, var='PSAL')
+    tableO = fill_tableqc(tableS,ds, var='DOXY')
+    tableC = fill_tableqc(tableO,ds, var='CHLA')
+
+    colrs = np.copy(tableC)
+    colrs[np.where(colrs != 'X')] = 'w'
+    colrs[np.where(colrs == 'X')] = 'lightcoral'
+
+    tab2_ax = fig.add_axes([0.33, 0.22, 0.25, 0.4], anchor='NE')
+    fig.patch.set_visible(False)
+    tab2_ax.axis('off')
+    tab2_ax.axis('tight')
+    table2 = tab2_ax.table(cellText=df2.values, colLabels=df2.columns, cellColours=colrs, cellLoc='center')
+    table2.scale(3, 2)
+    # table2.set_fontsize(104)
+
+    for (row, col), cell in table2.get_celld().items():
+        if (row == 0) or (col == -1):
+            cell.set_text_props(fontproperties=FontProperties(weight='bold', size=55))
+        if col == 0:
+            cell.set_text_props(fontproperties=FontProperties(weight='bold'))
+
+    ## Add logo at the bottom
+    im = plt.imread('logos.png')
+    newax = fig.add_axes([0.88, -0.03, 0.1, 0.1], anchor='NE')
+    newax.imshow(im)
+    newax.axis('off')
+    ax.text(0.78, -0.11, 'Created with Glidertest', transform=ax.transAxes, fontsize=font_size - 3,
+            verticalalignment='top')
+
+    # Add glider track plot
+    gt_ax = fig.add_axes([0.47, 0.64, 0.38, 0.25], projection=ccrs.PlateCarree(), anchor='SE', zorder=-1, )
+    plots.plot_glider_track(ds, ax=gt_ax)
+    # Add basic variables profiles plot
+    bv1_ax = fig.add_axes([0.5, 0.31, 0.21, 0.21], anchor='SE', zorder=-1)
+    bv2_ax = fig.add_axes([0.78, 0.31, 0.21, 0.21], anchor='SE', zorder=-1)
+    plots.plot_basic_vars(ds, v_res=1, start_prof=0, end_prof=-1, ax=[bv1_ax, bv2_ax])
+    fig.savefig(save_dir)
```

Comment:
could be nice to give it a useful name like SEA{glider_num}_M{mission_num}_report_{todays_date}.pdf